static/app/components/workflowEngine/simpleTable/index.spec.tsx

```diff
@@ -7,19 +7,19 @@ describe('SimpleTable component', function () {
     render(
       <SimpleTable>
         <SimpleTable.Header>
-          <SimpleTable.HeaderCell name="a">A</SimpleTable.HeaderCell>
-          <SimpleTable.HeaderCell name="b">B</SimpleTable.HeaderCell>
-          <SimpleTable.HeaderCell name="c">C</SimpleTable.HeaderCell>
+          <SimpleTable.HeaderCell>A</SimpleTable.HeaderCell>
+          <SimpleTable.HeaderCell>B</SimpleTable.HeaderCell>
+          <SimpleTable.HeaderCell>C</SimpleTable.HeaderCell>
         </SimpleTable.Header>
         <SimpleTable.Row data-test-id="row-1">
-          <SimpleTable.RowCell name="a">0</SimpleTable.RowCell>
-          <SimpleTable.RowCell name="b">1</SimpleTable.RowCell>
-          <SimpleTable.RowCell name="c">2</SimpleTable.RowCell>
+          <SimpleTable.RowCell>0</SimpleTable.RowCell>
+          <SimpleTable.RowCell>1</SimpleTable.RowCell>
+          <SimpleTable.RowCell>2</SimpleTable.RowCell>
         </SimpleTable.Row>
         <SimpleTable.Row data-test-id="row-2">
-          <SimpleTable.RowCell name="a">3</SimpleTable.RowCell>
-          <SimpleTable.RowCell name="b">4</SimpleTable.RowCell>
-          <SimpleTable.RowCell name="c">5</SimpleTable.RowCell>
```

Comment:
we never asserted on these `name` fields before, the test below does `getByRole('cell', {name: '0'})`